### DISCUSSION TITLE
importing the relevant modules in a suggestion

### DISCUSSION THREAD

**Discussion by m11ad** [community/community#7260] [Copilot Conversations]
Hi , 
I just started using the Copilot and it's really cool . But it'd be cooler when you choose a suggestion, it automatically imports the necessary modules too. 
for example in the python code bellow: 
`#print the current time and date`
`print(time.strftime("%H:%M:%S on %d/%m/%Y"))`

after running the file I get a _NameError : name 'time' is not defined_.

**Comment by cristian357r4** [comment]
but actually most IDEs and code editors can do it. I don't think it's a feature that copilot needs
